client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-table/style.scss

```diff
@@ -0,0 +1,94 @@
+
+@import '@wordpress/base-styles/_breakpoints.scss';
+@import '@wordpress/base-styles/_mixins.scss';
+
+.site-table {
+	&__table {
+		display: none;
+		@include break-large() {
+			display: table;
+		}
+		border: 1px solid var( --studio-gray-5 );
+		border-collapse: collapse;
+		tr {
+			height: 1px;
+			background: var( --studio-white );
+		}
+		th {
+			font-size: 0.75rem;
+			color: var( --studio-gray-50 );
+			font-weight: 400;
+			height: 40px;
+		}
+		td {
+			font-size: 0.875rem;
+			height: 50px;
+			width: 162px;
+			box-sizing: border-box;
+			&:first-child {
+				position: relative;
+				width: initial;
+				@include break-large() {
+					max-width: 200px;
+				}
+				@include break-huge() {
+					max-width: 300px;
+				}
+			}
+		}
+		td, th {
+			border-bottom: 1px solid var( --studio-gray-5 );
+			text-align: left;
+			border-collapse: collapse;
+			vertical-align: middle;
+			@include break-large() {
+				padding: 0 8px;
+			}
+			@include break-wide() {
+				padding: 0 16px;
+			}
+		}
+	}
+	&__inline-flex {
+		display: inline-flex;
+		width: 100%;
+	}
```

Comment:
Is this being used somewhere in the code? I couldn't find an HTML element with that class.